proto/framespace/framespace.proto

```diff
@@ -71,20 +61,5 @@ message DataFrame {
 	Dimension minor = 3;
 	repeated Unit units = 4;
 	map<string, string> metadata = 5;
-	repeated Vector contents = 6;
-}
-
-/** Special key: value map 
-* This is one cell in a DataFrame (an element of vector content list). 
-* value type should be enforced based on DataFrame.units
-**/
-message KeyValue {
-	string key = 1;
-	oneof value {
-		float float_data = 2;
-		string str_data = 3;
-		bool bool_data = 4;
-		int32 int_data = 5; 
-		sint32 sint_data = 6;
-	}
+	map<string, google.protobuf.Struct> contents = 6;
```

Comment:
Because this is essentially an 'Any' statement, can we include an example of a correct structure in the comments?